save-cloud-common/build.gradle.kts

```diff
@@ -44,10 +44,12 @@ kotlin {
             dependencies {
                 implementation(project.dependencies.platform(libs.spring.boot.dependencies))
                 implementation(libs.spring.security.core)
+                implementation(libs.spring.web)
```

Comment:
What does this dependency provide?
Edit: OK, found, it's for common methods to get HTTP statuses